cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,104 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  let student;
+
   before(() => {
+    student = {
+      firstName: generateRandomString(),
+      lastName: generateRandomString(),
+      email: generateRandomEmail(),
+      mobileNumber: generateRandomPhoneNumber(),
+      address: generateRandomAddress(),
+      gender: getRandomGender(),
+      hobby: getRandomHobby()
+    };
 
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  function generateRandomString() {
+    return Math.random().toString(36).substring(7);
+  }
+
+  function generateRandomEmail() {
+    return generateRandomString() + '@gmail.com';
+  }
+
+  function generateRandomPhoneNumber() {
+    return Math.floor(Math.random() * 10000000000).toString();
+  }
+
+  function generateRandomAddress() {
+    return (
+      Math.floor(Math.random() * 1000) +
+      ' ' +
+      generateRandomString() +
+      ' St, ' +
+      generateRandomString() +
+      ' City, ' +
+      generateRandomString() +
+      ' Country'
+    );
+  }
+
+  function getRandomGender() {
+    const genders = ['Male', 'Female', 'Other'];
+    return genders[Math.floor(Math.random() * genders.length)];
+  }
+
+  function getRandomHobby() {
+    const hobbies = ['Sports', 'Reading', 'Music'];
+    return hobbies[Math.floor(Math.random() * hobbies.length)];
+  }
```

Comment:
```suggestion
```
1. Support function should not take place in the test file, move them to support folder
2. Use faker for generating random data

Reply:
I tried to install the faker but couldn't (